numpy_challenge.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+if __name__ == "__main__":
+    array_1 = np.zeros(10)
+    array_2 = np.arange(10)
+    array_3 = np.random.rand(10)
+
+
+def matrix_multiplication(matrix_1, matrix_2):
+    return np.matmul(matrix_1, matrix_2)
+
+
+def multiplication_check(list_of_matrices):
+    multiplication_capability = True
+    for i in range(len(list_of_matrices) - 1):
+        rows_count_1, columns_count_1 = list_of_matrices[i].shape
+        rows_count_2, columns_count_2 = list_of_matrices[i + 1].shape
+        if columns_count_1 != rows_count_2:
+            multiplication_capability = False
+            break
+    return multiplication_capability

```

Comment:
Норм, можно чуток упростить
```suggestion
def multiplication_check(list_of_matrices):
    for i in range(len(list_of_matrices) - 1):
        rows_count_1, columns_count_1 = list_of_matrices[i].shape
        rows_count_2, columns_count_2 = list_of_matrices[i + 1].shape
        if columns_count_1 != rows_count_2:
            return False
    return True

```